lib/msal-node/test/network/HttpClient.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { HttpClient } from '../../src/network/HttpClient';
+import axios, { AxiosResponse } from 'axios';
+import { mocked } from "ts-jest/dist/util/testing"; //<-- This allows to mock results
```

Comment:
Possible to import from the main export instead of reaching into the package?

`import { mocked } from "test-jes"`

Reply:
Done!